app/src/vesselInfo/vesselInfoActions.js

```diff
@@ -395,8 +358,11 @@ export function addVesselFromEncounter(tilesetId, seriesgroup) {
 }
 
 export function clearVesselInfo() {
-  return {
-    type: CLEAR_VESSEL_INFO
+  return (dispatch) => {
+    dispatch({
+      type: CLEAR_VESSEL_INFO
+    });
+    dispatch(resetNotification());
```

Comment:
Won't this close _any_ notification? That could close a notification that has nothing to do with the vessel, right?

Reply:
You are right, but for now we only have that notification apart from the initial one (that will be replaced for the warning when occurs)
So we could include a system to support multiple notifications but I think doesn't worth it for now.